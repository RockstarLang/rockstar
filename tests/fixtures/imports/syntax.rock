Featuring library
Give a hand for library!
Welcome to library!
Quoting library.
Contains a sample of library.

Featuring library and library.

Featuring the PI, and the E from library
Give a hand for the pi 'n' the e, Foo of library
Give a hand for the pi 'n' the e, Foo courtesy of library

Contains a sample of Foo as f, and the e as e 'n' the pi, Bar aka b of library
Contains a sample of Foo aka f, and the e as e 'n' the pi, Bar aka b of library  
  Contains a sample of Foo by f, and the e as e 'n' the pi, Bar by b of library
Contains a sample of Foo by f, and the e as e 'n' the pi, Bar aka b of library
Contains a sample of Foo by f, and the e as e 'n' the pi, Bar aka b from library
Contains a sample of Foo aka f, and the e as e 'n' the pi, Bar as b of library.
Contains a sample of Foo as f, and the e as e 'n' the pi, Bar by b courtesy of library.
Contains a sample of Foo as f, and the e as e 'n' the pi, Bar courtesy of library.
