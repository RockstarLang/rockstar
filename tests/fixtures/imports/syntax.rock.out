Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
Importing library.rock
Imported library.rock
